src/plugins/flexsearch.js

```diff
@@ -22,20 +21,22 @@ function formatSearch({ node, root }) {
   const nodeString = stripBadge(toString(node));
 
   switch (nodeString) {
-  case 'Attributes':
+  case 'Attributes': {
```

Comment:
Change: We should not use opening and closing brackets for `case` statements - other repos don't have these

Reply:
@Ray2126 The only reason they are here because the by default the new eslint version doesn't allow to declare variables inside case blocks which is why these are wrapped in brackets.

https://eslint.org/docs/latest/rules/no-case-declarations